src/akeneo/harness/config/q-akeneo-pipeline.yml

```diff
@@ -0,0 +1,21 @@
+
```

Comment:
why q-akeneo-pipeline.yml?

Reply:
a before p before q I guess

Reply:
yes... spryker-pipeline.yml was nicer... but the right place in the alphabet